doc/reference/configuration/configuration_reference.rst

```diff
@@ -1028,6 +1030,33 @@ If :ref:`replication.failover <configuration_reference_replication_failover>` is
     | Default: nil
     | Environment variable: TT_CONFIG_ETCD_HTTP_REQUEST_UNIX_SOCKET
 
+.. _config_etcd_watchers_reconnect_max_attempts:
+
+.. confval:: config.etcd.watchers.reconnect_max_attempts
+
+    **Since:** :doc:`3.1.0 </release/3.1.0>`
+
+    The maximum number of attempts to try to reconnect to an etcd server in case of connection failure.
```

Comment:
```suggestion
    The maximum number of attempts to reconnect to an etcd server in case of connection failure.
```

Or maybe rephase using the `reconnection attempts` wording